examples/lorenz_attractor.ipynb

```diff
@@ -10,6 +10,12 @@
     "\n",
     "If you are looking at a static version of this notebook and would like to run its contents, head over to [GitHub](https://github.com/giotto-ai/giotto-tda/blob/master/examples/lorenz_attractor.ipynb) and download the source.\n",
     "\n",
+    "## See also\n",
+    "\n",
+    "- [Topology of time series example](https://giotto-ai.github.io/gtda-docs/latest/notebooks/topology_time_series.html), in which the *Takens embedding* technique is explained on simple examples and in detail.\n",
```

Comment:
As above, I suggest removing "example" from text in brackets. Also maybe a small wording suggestion: `in which the *Takens embedding* technique used here is explained in detail and illustrated via simple examples.`